### DISCUSSION TITLE
Securing branding

### DISCUSSION THREAD

**Discussion by salvo-uk** [OrchardCMS/OrchardCore#7688] [Q&A] [answered]
Hi,

Is it possible to secure Orchard such that core branding (colours, fonts, heading styles etc) cannot be changed by regular content authors? i.e. So they can only be changed by developers or designated 'approvers' / admins.

Thanks
Salvo

**Comment by nogginbox** [comment, marked as answer]
Normally the branding will be set by the theme. The built in text editors you can choose encourage using simple HTML or markdown that will be controlled by the theme's CSS.

Orchard has a rich and highly customiseable permission system. so you can decide which users have permission to change the active theme.
